src/index.test.ts

```diff
@@ -2268,3 +2289,54 @@ describe('Experimental options togglesStorageTTL enabled', () => {
         });
     });
 });
+
+describe('updateToggles', () => {
+    it('should not update toggles when not started', () => {
+        const config: IConfig = {
+            url: 'http://localhost/test',
+            clientKey: '12',
+            appName: 'web',
+        };
+        const client = new UnleashClient(config);
+
+        client.updateToggles();
+
+        expect(fetchMock).not.toHaveBeenCalled();
+    });
+
+    it('should update toggles when started', async () => {
+        const config: IConfig = {
+            url: 'http://localhost/test',
+            clientKey: '12',
+            appName: 'web',
+        };
+        const client = new UnleashClient(config);
+
+        await client.start();
+        fetchMock.mockClear();
+
+        client.updateToggles();
+
+        expect(fetchMock).toHaveBeenCalled();
+    });
+
+    it('should wait for readyness before update toggles', async () => {
```

Comment:
maybe 'should wait for client readiness before the toggles update'?

Reply:
Changed